piweatherrock/weather.py

```diff
@@ -107,8 +106,8 @@ def sizing(self, size):
 
         if self.config["fullscreen"]:
             self.screen = pygame.display.set_mode(size, pygame.FULLSCREEN)
-            self.xmax = pygame.display.Info().current_w - 35
-            self.ymax = pygame.display.Info().current_h - 5
+            self.xmax = pygame.display.Info().current_w #  - 35 Why not use full screen in "fullescreen"?
+            self.ymax = pygame.display.Info().current_h #  - 5 Why not use full screen in "fullescreen"?
```

Comment:
This is one of the legacy bits of code that I can't remember why its here. I will try it out w/o the extra space tonight

Reply:
It just displays a rather big black bar on my tiny tft screen. It's small as it is, without restricting it further :)